### PR TITLE
Fixed integration_test.go

### PR DESCRIPTION
**Pull request by debedb**
Agent was commented out from integration_test.go, made it work again

https://paninetworks.kanbanize.com/ctrl_board/3/cards/277/comments

### CODE REVIEW

**Comment by flashvoid** [comment]
+1

File: test/integration_test.go [hunk 1]
```diff
@@ -141,15 +144,30 @@ func (s *MySuite) TestIntegration(c *check.C) {
...
+	// Find some romana IPs that we can use... Because the agent checks for those
+	addrs, err := net.InterfaceAddrs()
+	if err != nil {
+		c.Error(err)
+	}
+
+	possibleRomanaIps := make([]string, 0)
+	for _, addr := range addrs {
+		strAddr := addr.String()
+		if strings.ContainsAny(strAddr, ":") {
```
Comment (jbrendel) [inline test/integration_test.go]: What does the presence of a ':' signify and why do we ignore that one? Maybe a little comment to explain that.

File: test/integration_test.go [hunk 2]
```diff
@@ -141,15 +144,30 @@ func (s *MySuite) TestIntegration(c *check.C) {
...
+	myLog(c, "Possible RomanaIPs:", possibleRomanaIps)
 	host1 := common.HostMessage{}
 	client.Post(hostsRelURL, newHostReq, &host1)
 	myLog(c, "Response: ", host1)
 	c.Assert(host1.Ip, check.Equals, "10.10.10.10")
 	c.Assert(host1.Id, check.Equals, "1")
-	//
-	newHostReq = common.HostMessage{Ip: "10.10.10.11", RomanaIp: "10.0.0.2/16", AgentPort: 9999, Name: "HOST2000"}
+
+	newHostReq = common.HostMessage{Ip: "10.10.10.11", RomanaIp: possibleRomanaIps[1], AgentPort: 9999, Name: "HOST2000"}
```
Comment (jbrendel) [inline test/integration_test.go]: Is there ever a chance that we will get less than two possible IPs?

Comment (debedb) [inline test/integration_test.go]: There is but this is a test... Do we bother?

File: test/integration_test.go (hunk 1 above)
Comment (debedb) [inline test/integration_test.go]: Will add a comment that I'm ignoring IPv6 stuff.

File: test/integration_test.go (hunk 2 above)
Comment (jbrendel) [inline test/integration_test.go]: If we can guarantee that during a test we have at least two then it's fine.

**Pull request merged**